### PR TITLE
proposed changes in preparation for 5/1/16 discussion

### PR DESCRIPTION
**Pull request by henryhund**

### CODE REVIEW

File: norms.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+##Fishtown Analytics Norms##
+
+###Listen, then speak.###
```
Comment (jthandy) [inline norms.md]: This is the only thing you've written so far that I don't actually agree with. I think "think, talk, think" and "talk, think, talk" are both personal styles and ones that we should not attempt to create norms around. I do agree with the first sentence of this paragraph though.

File: norms.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+##Fishtown Analytics Norms##
```
Comment (jthandy) [inline norms.md]: I need your help understanding what the difference between a norm and a value is. It seems like you're heavily borrowing from what I had listed as values, so it seems like there is some overlap in our conceptions.

I'm also working on something I'm calling the "company operating system" which I had imagined overlapping with norms. It's a bit more specific than simply a list, however.

In general, I just think we should define the structure of this activity when we get together. I'm thinking through it as I go (as I'm sure are you) so I'm sure it's all productive, we just need to figure out how we're getting to some type of state we're happy with.

File: norms.md (hunk 1 above)
Comment (henryhund) [inline norms.md]: Maybe a better way to describe what I'm trying to communicate is "Listen well." Or "Commit to understanding."

File: norms.md (hunk 2 above)
Comment (henryhund) [inline norms.md]: Maybe the difference is meaningless. But I generally think of it like this:

Values describe what you aspire to be
Norms describe how you act so you can achieve these values

Right now, the values and norms described (as I've divided them out) may not perfectly adhere to the above. And maybe it isn't a distinction worth making, anyway. You're right we should probably discuss a bit more in person.

**Comment by drewbanin** [comment]
let's meditate on this for another year. till then, closing

**Pull request closed without merging**